### DISCUSSION TITLE
Updating a page in VSCode switches all open tabs to that page

### DISCUSSION THREAD

**Discussion by lincmad** [yandeu/five-server#89] [Help / Questions]
I have been very pleased with Five Server, but there is one feature that I find quite annoying.

Let’s say I’m working on a project where I need to update multiple pages. I open each page via Five Server in a separate browser tab. I can flip easily from one page to another. When I update a page in VSCode, the Five Server automatically refreshes the browser. So far, so good. However, Five Server also switches **_every open tab_** to the specific page I just edited. 

This is undesirable behavior. I want to update the tab for the page that I just changed, but leave the other tabs pointed to the various other pages. It’s fine to have the browser refresh all the tabs, but don’t change which pages they’re pointing to.

**Comment by yandeu** [comment]
This feature is called `navigate`. You can simply disable it in the extension settings.

**Reply by bhaveshxrawat** [reply]
Hey, @yandeu! I just turned off the navigate feature in the settings, but it turns out that it turns off the navigation feature completely for all tabs and not just the tabs that are on `blur` state. OP is right! updating every tab is not desirable. Can you make this change or maybe add it as an additional checkbox?

**Reply by yandeu** [reply]
You mean only the "active" tab should change if "navigate" is on?

**Reply by bhaveshxrawat** [reply]
Yes! Exactly that! Would it be hard to do?
